src/Provider/Provider.Web/Configuration/EsfaCookieOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Esfa.Recruit.Vacancies.Client.Application.Providers;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+
+namespace Esfa.Recruit.Provider.Web.Configuration
+{
+    public static class EsfaCookieOptions
```

Comment:
Could we move this to the RCL project? 👍 on the rest of the items moved to the RCL project.